tools/ci/testlist/sim-02.dat

```diff
@@ -33,6 +33,16 @@
 # macOS matter compilation is not currently supported
 -Darwin,sim:matter
 
+# macOS doesn't support cxxabi.h
+-Darwin,sim:libcxxtest
```

Comment:
https://github.com/apache/nuttx/tree/master/libs/libxx/libcxxabi could provide cxxabi.h, can we switch to it?

Reply:
Sorry not quite sure how to switch it?

Reply:
I'll post the above as NuttX Issues, since I'm not able to resolve them. Thanks!

Reply:
I have created the issue: https://github.com/apache/nuttx/issues/14774